tests/mixins/responsiveManagerMixin-test.js

```diff
@@ -0,0 +1,63 @@
+jest.dontMock('../../js/mixins/responsiveManagerMixin');
+jest.dontMock('../../js/components/utils');
+
+const React = require('react');
+const Enzyme = require('enzyme');
+const debounce = require('lodash.debounce');
+const createReactClass = require('create-react-class');
+const responsiveManagerMixin = require('../../js/mixins/responsiveManagerMixin');
```

Comment:
Give a look on how it can be imported: https://github.com/ooyala/html5-skin/blob/master/tests/components/volumeControls-test.js

Reply:
I know about "import", thank you :-)